### ISSUE TITLE
Bonereaver's Edge procc is using the incorrect version for the current patch on Elysium

### ISSUE THREAD

**Issue by Cephel** [LightsHope/server#394]
http://wowwiki.wikia.com/wiki/Patch_1.10.0 clearly states: `The Bonereaver's Edge's triggered effect has been changed. Instead of lowering your victim's armor, it will cause you to ignore a portion of your victim's armor. Its triggering rate was substantially increased as well as the magnitude of the effect. In addition, the item now has 16 stamina. `

This refers to this version of the weapon: ![](https://i.imgur.com/2aqZaCZ.png), you can see a scan of the weapon data at https://web.archive.org/web/20060531205116/http://wow.allakhazam.com:80/item.html?witem=17076 which is from May 31st, 2006, which is after the release of 1.10.0 (release date: 28 March 2006)

Prior to 1.10, the weapon should have the following stats: ![](https://i.imgur.com/mortpw2.png)
Scan from https://web.archive.org/web/20051205110110/http://wow.allakhazam.com:80/item.html?witem=17076 which is from December 5, 2005.

Additionally, not only is the procc using the wrong version, it is additionally using the SUPER incorrect version that stacks up to 3 times. Here is a scan of the procc itself immediately prior to 1.11.0: https://web.archive.org/web/20060525082922/http://wow.allakhazam.com:80/db/spell.html?wspell=21153 which shows that by that time, the procc still did NOT stack to 3.

The **earliest possible patch** the 3 stacks could have been present is 1.11.0 thusly.

Since the effect of the procc makes this **the strongest DPS item in the game by far** right now, it is utterly detrimental to the proper gearing choices for fury warriors and should be changed back to a patch appropriate version. Something like 80% of the top DPS on Horde is using a BRE.

**Comment by Cephel** [comment]
If for whatever reason the proper patch version (300 armor debuff) cannot be restored, at the very least it should be brought down to not stacking to 3.

**Comment by ohganot** [comment]
Light's Hope uses 1.12.1 spell progression. Which means BRE 3 stacks is normal.

Besides, it isn't as overpowered as you think. On horde side, BiS orc warrior with axes will still beat a BRE warrior. On alliance side, BRE warriors will only beat undergeared dualwield warriors. 

Don't claim that 80% of the top DPS on Horde are using BRE. It's very easy to check for that using realmplayers and that number is more like 30%.

I don't think it's bad having overly strong weapons. That's part of vanilla, some items were very strong and some were very weak. For example, the current ironfoe with 6% proc chance is BiS for alliance till Kel'thuzad weapons and is about the same dps as Gressil, Dawn of Ruin. Hand of Justice drops from BRD and is a blue quality item and yet it is used up to the late stages of AQ40.

**Comment by Cephel** [comment]
**Light's Hope uses progressive itemization and should revert the weapon back to the proper state**. Quoting spell progression for an item is dubious at best. The reason we use 1.12 spells is because we would have to modify the client in order to change talent trees and base spells.

I asked in the other issue to list the items resulting in ~4000 armor reduction on the standard Boss armor and you failed to provide them so far. BRE with 3 stacks is insanely strong and has no business being this competetive when it never was this good in retail at **any** point during Vanilla.

Your Ironfoe comment is an indicator you propably should not be arguing on this topic. I went back and checked on some Naxx bosses that are easy to parse on for Anathema, **not one of the top 25 DPS on them uses Ironfoe**, not a single one.

**Comment by brotalnia** [comment]
All spells are in their 1.12.1 state as that is the client being used. The spell data is in the DBC files, and those come from the client. Even if you were to hack the data on the server side, it would look totally wrong in the client. The only solution is to either force everyone to use a custom MPQ patch, which means Warden would have to check for the existence of the file and kick anyone without it, or attempt to actually downgrade the server to the old patch and have people use the appropriate client. None of these are practical or possible to implement in a reasonable time. This is why spells are not changed. The progression applies only to database data. Item stats are in the database, the spell data is from the client.

**Comment by Cephel** [comment]
The item and its procc are interwtined and cannot be seperated, it's not the item itself that's the issue, it's the procc that it does. If the procc elevates the weapon from okay tier to replace at KT tier, then there is an issue here on a progressive server with raids not out.

This is why PvP gear is currently downgraded too, because it is equivalent and sometimes superior to AQ40 loot, however there is no way to get to AQ right now.

Please stop arguing semantics for something that is obviously very bad for the gear progression of the server.

Changing the stack count of the weapon is trivial, as it is just a parameter to the spell, rather than something that requires editing the client. You could make it stack to 500 too if you wanted.

**Comment by brotalnia** [comment]
That is what is called a hack. It would have to be manually done in `spell_effect_mod` every time the db is dumped and replaced by fresh one from development branch. This is how all progression worked before the new system and it was a total mess, cause nobody knew what is in the live server databases and most migrations would fail and not apply correctly due to differences in the databases. And it's also picking and choosing. The next day you'll have people saying you changed BRE spell, why can't you change that other spell, and before you know it the db is full of spell hacks that look wrong in the client. Either we move all DBC loading to the database and add tables for all previous patches, which means we have full spell progression, or we don't change any spells. These are the clean, properly done choices, without manual hacks on specific servers. But since implementing this will both take a very long time, and require people to use custom patches for spells to display properly, we go with the second option of not having spell progression.

**Comment by Aklol** [comment]
fyi, you're not seeing any players using ironfoe/anubsiath in naxx is because it's bis with world buffs. other than that, it's not, but it's close.

quite frankly, this BRE business is overblown. there's nothing wrong with a weapon being utterly bis to use for a while (e.g. ironfoe, which is pre-raid). especially when it drops in a raid as opposed to a level 55-60 instance. the difference between using BRE and equivalents are not huge, just as using ironfoe vs. cts/r14 is not huge.

**Comment by Cephel** [comment]
The effort can and should be done to fix this, because it completely destroys all gear progression for warriors.

Yes there is an issue from a weapon being bis when it wasn't bis in Vanilla, not even close.

And I'd pay a little more thought into this "well it doesn't matter" if the droprate on it used to be completely broken, with hundreds of them in circulation now, when it's supposed to be as rare as a binding.

**Comment by Cbabya** [comment]
First of all, the viewpoint that this is fine because of the premise that devs dont change spell effects to make the experience more blizzlike is completely indefensible, because they do. For an example: Spell oils for casters are disabled. 

Using the same logic that BRE should have its proc (=spell effect) changed so that the progression is more blizzlike, it was decided that spell oils should be unusable (=spell effect made unusable) so that progression is more blizzlike.
"They shouldn't do it because it's a spell effect and all spell effects are 1.12.1" is therefore not an argument because it's untrue. They arent and there's plenty of examples of them not being, with devs "jury-rigging" things to keep the experience blizzlike. 

Secondly, "it's not a big deal" @Aklol : How serious are you in saying that? BRE remains BiS literally until naxx, where it may be replaced with a Might of Menethil, a drop off KT, but generally it's not worth doing that because BRE comes out ontop in many situations. 
You're saying that a rag weapon drop being as strong as a KEL THUZAD WEAPON DROP (please let it sink in) in a patch where KEL THUZAD ISN'T EVEN CLOSE TO BEING OUT YET (again, let it sink in) is "just a bit of an inconsistency". This is about as big of a deal as it could be, as far as misitemization is concerned. 

TL;DR "devs dont change spell effects" they do
TL;DR "it's not a big deal" it is

It was proven by a previous change that it's very possible to simply limit the stacks of the buff, which, while not blizzlike, made it perform similiarly in progression, which is what the developer team has stated to be their goal: Not blizzlike, but blizzlike experience. I hope they'll continue to honor that.

**Comment by brotalnia** [comment]
Disabling spells is not the same as changing them.

**Comment by Cbabya** [comment]
I don't see how that affects any of the points made, if it's supposed to.

**Comment by knasen88** [comment]
and onslaught girdle is bis from day 1 til release of fresh server

**Comment by Cbabya** [comment]
@knasen88 Partially true, there's an upgrade in naxx. 
 
Irrelevant either way because this item progression is blizzlike. BRE being BiS every single patch up to naxx patch is however not.

**Comment by Aklol** [comment]
the best gloves in the game for warriors using swords or axes are a level 44 world drop epic. i think a ragnaros super rare drop being the best item until naxx isn't so bad. i agree the proc is incorrect for the patch, however, i think this issue is overblown.

**Comment by Cbabya** [comment]
These relativizations make very little sense. The fact that edgemasters is BiS until AQ40 which is the way it was on retail does not change at all the fact that BRE isn't supposed to be as strong as it is now. 

You do understand that this change means that when an ashkandi drops from nef in 1.7 and you have BRE, you pass on it, right? 
AQ40 weapon drops in the AQ patch? you fucking PASS ON EM all because you have BRE. Can you imagine this? 
Did ANYONE do this on retail outside of the naxx patch?

This isn't one bit a blizzlike experience and as such should be changed. It doesn't matter how subjectively silly or badly balanced item progression is according to you, fact of the matter is that this isn't blizzlike.

**Comment by Aklol** [comment]
what's irrelevant if anything is comparing what's happening now to what happened on retail. people didn't know how to gear and itemize properly until the end of naxx.

again, BRE is not a 'must have'. while it is arguably the best, on horde side, ashkandi is very comparable as 2h fury, and db/crul as 1h fury, especially when most top tier raids are already using sunder/faerie fire/annihilator/yield on bosses.

**Comment by Cbabya** [comment]
> what's irrelevant if anything is comparing what's happening now to what happened on retail. people didn't know how to gear and itemize properly

Coincidentally, that's also why people didn't raid. Only a small amount of users took the game seriously enough to theorycraft and figure out what to wear. 

You're heavily misrepresenting the active raiding community back then. Although small, there was an insane amount of theorycrafting going on, and things like BRE being this broken would've been abused.

**Comment by Cephel** [comment]
>again, BRE is not a 'must have'. while it is arguably the best, on horde side, ashkandi is very comparable as 2h fury

Now this is just patently false. The armor debuffs you listed are not enough to reduce a boss down to 0 or close to it. The remaining armor accounts for up to 15% more damage from BRE at 3 stacks on an ideal boss (C'thun for example). Nothing in the game comes close to BRE for a 2h fury, MoM is **barely** better.

**Comment by ohganot** [comment]
Having Sunder Armor, Curse of Recklessness, Faerie Fire and Annihilator up means -3995 armor reduction which results in a normal boss with 4691 armor being reduced to 696 armor. Which means BRE only benefits from one stack.

**Comment by GuybrushGit** [comment]
As Brotalnia said, hackfixing w/e spells we feel like it is the wrong solution and will lead to the same database issues we had on Elysium and I don't think we have the resources to implement full spell progression.

There are 3 options: keep BRE as it is, create a custom patch to disable the proc pre 1.10, create a custom patch to replace the BRE proc with another less overpowered spell pre 1.10.

**Comment by ohganot** [comment]
Nerfing it to 1 stack will cause almost no difference for hardcore guilds who use all armor debuffs, since boss will benefit only from 1 stack. Even most casual guilds use atleast Sunder Armor, Curse of Recklessness, Faerie Fire which means boss has 1296 armor therefore BRE only benefits from 1.85 stacks.

A 1 stack BRE is still stronger than every 2h weapon until Naxxramas weapons, so nerfing it to 1 stack doesn't change it.

Orc warriors will always do more dps with axes than BRE, main BRE users on horde side are non-orcs for whom it is BiS until Naxxramas weapons. On Alliance side, nobody really uses BRE unless they don't have raid level dual-wield weapons.

You can look these stats up on Realmplayers and you can also use various warrior spreadsheets on the forums the look up the theorycrafting. Stop spreading false info.

**Comment by bgfds** [comment]
Not one comment on how the weapon is "broken" in PvP even more so than in raids
Interesting
